linux_os/guide/system/network/network-ipsec/package_openswan_installed.rule

```diff
@@ -0,0 +1,35 @@
+documentation_complete: true
+
```

Comment:
Let's make this all libreswan. Openswan was deprecated a long time ago.

Reply:
@redhatrises Okay; I removed this one. `package_libreswan_installed.rule` already exists.